### PR TITLE
[MIG] internal_transfer_with_agreed_amount: Migration to 13.0

### PR DESCRIPTION
**Pull request by luiseevaquer**

### CODE REVIEW

**Comment by hugho-ad** [comment]
LGTM :+1: 

@edy1192 @luisg123v 

could you review please

File: internal_transfer_with_agreed_amount/i18n/es.po [hunk 1]
```diff
@@ -0,0 +1,112 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+#	* internal_transfer_with_agreed_amount
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 12.0+e\n"
```
Comment (luisg123v) [inline internal_transfer_with_agreed_amount/i18n/es.po]: Update version

```suggestion
"Project-Id-Version: Odoo Server 13.0+e\n"
```

**Review by luisg123v — APPROVED** [review]
In general LGTM, only one small comment

**Comment by edy1192** [comment]
Hi @lescobarvx @hugho-ad Could you do a test instance? To give functional approval.

Regards.

cc @luisg123v

**Review by luisg123v — APPROVED** [review]

**Comment by edy1192** [comment]
Hi @lescobarvx @hugho-ad  Functional tests :+1:

**Pull request merged**